File: problemas/strings/2137/2137.py
def main():
    while True:
        try:
            N = int(input())
            livros = []
            for _ in range(N):
                
                codigo = input()
                livros.append(codigo)
            
            livros.sort()

            for i in range(N):
                print(livros[i])
        except EOFError:
                break

if __name__ == "__main__":
    main()

File: problemas/strings/2137/2137.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input: string = "";
let lineCount: number = 0;

process.stdin.on("data", (chunk: string) => {
    input += chunk;
});

process.stdin.on("end", () => {
    let lines: string[] = input.split('\n');

    while (lines.length) {
        let N: number = parseInt(lines.shift() as string);
        let livros: string[] = [];

        for (let i = 0; i < N; ++i) {
            let codigo: string = lines.shift() as string;
            livros.push(codigo)
        }

        livros.sort()

        for (let j = 0; j < N; ++j) {
            console.log(livros[j]);
        }
    }
});
